src/asciidoc/docs/develop/app-dev-process.adoc

```diff
@@ -0,0 +1,204 @@
++
+Learn more: <<entitlements#>>
+
+=== Add @withkoji/core to the frontend
+
+Follow these steps to use the @withkoji/core package in the frontend of your app.
+
+[TIP]
+====
+Before you start developing your code, check for outdated dependencies in the forked app.
+
+* If the forked app uses deprecated Koji packages, migrate your project to use the @withkoji/core package.
++
+ Learn more: <<migrate-koji-core#_do_i_need_to_migrate, Do I need to migrate?>>
+
+* Check for other outdated dependencies in the forked app, and update them if necessary.
++
+ Learn more: <<work-locally#_installing_the_dependencies, Installing dependencies>>
+====
+
+. In your frontend directory, install or update the @withkoji/core package.
++
+[source, Bash]
+npm install @withkoji/core
+
+. In your frontend code, import the package.
++
+[source,Javascript]
+import Koji from '@withkoji/core';
+
+. Use the `Koji.config` function to initialize the package with your app's configuration data before any data in the application is rendered.
+For example:
++
+[source,Javascript]
+----
+// Initialize
+Koji.config(require('././koji.json'));
+
+// render
+render();
+----
+
+. Use the `Koji.ready` function to indicate that the application is ready to start receiving events.
+
+. Implement the desired core functionality.
+For example, you can provide customization options so creators can add their own content and branding to the app before adding it to their profiles.
++
+Learn more: <<core-frontend-overview#>>, <<capture-user-input#>>, <<thumbnail-image#>>
+
+=== Add @withkoji/core to the backend
+
+If your app has a backend service, follow these steps to use the @withkoji/core package in it.
+
+. In your backend directory, install or update the @withkoji/core package.
++
+[source, Bash]
+npm install @withkoji/core
+
+. In your backend code, import the package and your app's configuration data.
++
+[source,Javascript]
+----
+import { KojiBackend } from '@withkoji/core';
+
+import kojiConfig from '../../koji.json';
+----
+
++
+TIP: To initialize the core package in your backend, the `koji.json` file must be bundled with the backend service during deployment of your app.
+Many frameworks won't bundle this file by default because it is not in the same folder as the backend code.
+To solve this problem, you can use a tool such as https://www.npmjs.com/package/babel-plugin-inline-json-import[babel-plugin-inline-json-import].
+
+. Use `KojiBackend.middleware` to initialize the package with your app's configuration data.
++
+The middleware manages the scope for backend functionality, so that your code can support your initial app along with all of the custom versions.
+It works by parsing the request headers that the Koji platform uses to pass version-specific data to the backend.
++
+You must apply this middleware before handling any routes.
+The middleware will transform `res.locals`, and then modules in KojiBackend will accept `res` as a configuration parameter.
```

Comment:
```suggestion
The middleware will transform `res.locals`, and modules in KojiBackend will accept `res` as a configuration parameter.
```

Is KojiBackend a class? If so, should it be between backticks?

Reply:
It probably should. I added the backticks here. There are probably some more places in other docs where they needed to be added. Something to look for when you update other pages.